Chapter3/Chapter3.sln

```diff
@@ -0,0 +1,61 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 16
+VisualStudioVersion = 16.0.29519.87
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Chapter3.Task1", "Chapter3.Task1\Chapter3.Task1.csproj", "{C4E24C30-74CC-4D55-893C-A7B911116A10}"
+EndProject
```

Comment:
Why have you added so many projects to solution file? I expect to see here only Chapter3.Task1